File: C#/Dappe_ORM/DAL/Repositories/EmployeeRepository.cs
using DAL.Repositories;
using DAL.Entities;
namespace DAL.Repositories;

public class EmployeeRepository : GenericRepository<Employee, int>
{
    public EmployeeRepository(string connectionString) : base(connectionString, "Eployees", "EmployeeID")
    {
    }
}

File: Node/TypeORM_ORM/src/dal/repositories/employeeRepository.ts
import { AppDataSource } from '../../config/data-source';
import { Employees } from '../entities/Employees';
import { GenericRepository } from './genericRepositroy';

export class EmployeeRepository extends GenericRepository<Employees, number> {
  private static repo = AppDataSource.getRepository(Employees);
  constructor() {
    super(EmployeeRepository.repo, 'employeeId');
  }
}
